modules/loader-utils/src/lib/request-utils/request-scheduler.js

```diff
@@ -102,6 +102,9 @@ export default class RequestScheduler {
 
   // Refresh all requests and
   _issueNewRequestsAsync() {
+    if (!this._updateNeeded) {
```

Comment:
Yes, this makes sense... 

Actually, looking at line 100, it looks like we could have a lot of timeouts outstanding. Maybe save the timerId, it should be enough to just make sure we have one pending timer to call this function whenever something changes.